src/components/TabsPanel.tsx

```diff
@@ -1,12 +1,10 @@
 import React from "react";
-import { ListingItem } from "./ListingItem";
-import { useSelector, useDispatch } from "react-redux";
-import { ApplicationState } from "../app/store";
-import { Tab as TabPanel, Tabs } from "@material-ui/core";
+import { Tab as TabInPanel, Tabs } from "@material-ui/core";
```

Comment:
Why this alias?

Reply:
No reason, thanks for pointing that out.